<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1506</width>
    <height>926</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>1631</width>
      <height>971</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Empleados</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Nombre* : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_nameInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Apellido : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_lastnameInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ID* : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_idInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_birthInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Año de Nacimiento :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Año de Ingreso* : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_entryInput">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addEmployeeBT">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>310</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 100%;
heigth: 100%;</string>
      </property>
      <property name="title">
       <string>Tipo de Empleado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <widget class="QRadioButton" name="radioCommision">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Comision</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioSalaried">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Asalariado</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AGREGAR EMPLEADO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>70</y>
        <width>651</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AGREGAR CLIENTE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>320</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Nombre : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>350</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Monto* :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="c_idInput">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>290</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="c_nameInput">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>320</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="c_amountInput">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>350</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>290</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ID* : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboNameEmployee">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>160</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Nombre del Empleado</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
      </property>
      <property name="minimumContentsLength">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Nombre del Empleado</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboNumOfYear">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Años de antiguedad</string>
      </property>
      <item>
       <property name="text">
        <string>Años de antiguedad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Menos de 2 años</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 - 3 años</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 - 7 años</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8 - 15 años</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Más de 15 años</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="cant_client">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>200</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="message">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>510</y>
        <width>1500</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>220</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>50</y>
        <width>20</width>
        <height>441</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addClientBT">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>420</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Datos</string>
     </attribute>
     <widget class="QPushButton" name="deleteBT">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>150</y>
        <width>91</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showAllBT">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>150</y>
        <width>111</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar Todos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="employeeMoreSalaryBT">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>150</y>
        <width>191</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Empleado con mayor salario</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchBT">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>150</y>
        <width>81</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchInput">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>150</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="employeeMoreClientBT">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>191</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Empleado con mas clientes</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="EmployeeTable">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>1271</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <row>
       <property name="text">
        <string>0</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Año de Nacimiento</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Año de Ingreso</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tipo de Empleado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Salario</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Clientes</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>qwedas</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>jgv</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>jh</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>bhbj</string>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>kbh</string>
       </property>
      </item>
      <item row="0" column="6">
       <property name="text">
        <string>kbh</string>
       </property>
      </item>
      <item row="0" column="7">
       <property name="text">
        <string>jkb</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>150</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>1211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Datos de los Empleados</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="totalPayrollInput">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>570</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>560</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioPayrollSalaried">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nomina Asalariado</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioPayrollComission">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>10</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nomina Comision</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="clientTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>640</y>
        <width>551</width>
        <height>192</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>180</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>44</number>
      </attribute>
      <column>
       <property name="text">
        <string>Identificación</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre completo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>monto recibido</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>620</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Clientes del empleado selecionado</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1506</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEmpleado">
    <property name="title">
     <string>Empleado</string>
    </property>
    <addaction name="actionEmployeeView"/>
    <addaction name="actionInfoView"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuEmpleado"/>
  </widget>
  <action name="actionAgregar">
   <property name="text">
    <string>Agregar</string>
   </property>
  </action>
  <action name="actionConsultar">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionEmployeeView">
   <property name="text">
    <string>Agregar</string>
   </property>
  </action>
  <action name="actionInfoView">
   <property name="text">
    <string>Listado de empleados</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
